atom.xml

```diff
@@ -22,7 +22,7 @@
     <published>{{ post.date.toISOString() }}</published>
     <updated>{{ post.updated.toISOString() }}</updated>
     {% if config.feed.content and post.content %}
-    <content type="html"><![CDATA[{{ post.content | safe }}]]></content>
+    <content type="html"><![CDATA[{{ post.content | uriencode }}]]></content>
```

Comment:
It sucks when the post's content contains emoji chars like 😀
Use the `uriencode` filter can resolve this problem.
https://github.com/o2team/o2team.github.io/issues/21